OneCommunityGlobal/HighestGoodNetworkApp
# Lawrence - Hide sensitive data hotfix

Description

![Task desc](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46981223/238eec9b-0275-4fbc-997b-883b10657122)

Main changes explained:
Hide hardcoded login credentials using the .env file for creating a new user.
How to test:
check into current branch
do npm run start:local to run this PR locally
log as admin user
go to Other Links→ User Management
press Create New User button and create a new user
verify that the user is created successfully which means the environment variable was used successfully.
